.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "rules": {
+    "semi": [2, "never"],
+    "prefer-arrow-callback": 0,
+    "space-before-function-paren": [2, "never"],
+    "func-names": 0,
+    "no-param-reassign": 0,
```

Comment:
Mainly to make optional parameters work, this is what we do now
```js
function a(b, c) {
  c = typeof c === 'string' ? c : 'something'
}
```
We'll have to do this instead
```js
function a(b, cGiven) {
  const c = typeof cGiven === 'string' ? cGiven : 'something'
}
```

I personally prefer the first one